### PR TITLE
2022.8.0

### PR DESCRIPTION
**Pull request by frenck**
# ⚠️ Draft PR for reviewing purposes. Please do not merge!

Feel free to review the release notes (or anything else in this PR)! ❤️ 

Still in this PR:

- Beta warning
- Incorrect date

Both need to removed/adjusted before release of course.

### CODE REVIEW

File: source/_posts/2022-08-03-release-20228.markdown [hunk 1]
```diff
@@ -0,0 +1,568 @@
...
+to adapt them to this change.
+
+([@eifinger] - [#72405]) ([documentation](/integrations/here_travel_time))
+
+[@eifinger]: https://github.com/eifinger
+[#72405]: https://github.com/home-assistant/core/pull/72405
+
+{% enddetails %}
+
+{% details "HomeKit Controler" %}
```
Comment (Jc2k) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
{% details "HomeKit Controller" %}
```

Comment (frenck) [reply source/_posts/2022-08-03-release-20228.markdown]: Fixed! Thanks!

File: source/_posts/2022-08-03-release-20228.markdown [hunk 2]
```diff
@@ -0,0 +1,560 @@
+---
+layout: post
+title: "2022.8: You can fix it!"
+description: "First-class support for Bluetooth, get notified about detected issues and how to repair them, find integrations by brands name, and maps with much better colors."
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
description: "First-class support for Bluetooth, get notified about detected issues and how to repair them, find integrations by brand name, and maps with much better colors."
```

**Review by pvizeli — DISMISSED** [review]

File: source/_posts/2022-08-03-release-20228.markdown [hunk 3]
```diff
@@ -0,0 +1,560 @@
...
+A brand new [Bluetooth integration] has been added, which provides absolutely
+amazing Bluetooth support to Home Assistant, including automatic discovery of
+new devices and the ability to push device updates.
+
+Integrations that implement device support can easily be built on top of these
+features, meaning we now have a solid base in Home Assistant to unlock
+support for many Bluetooth devices out there!
+
+The screenshot below shows that the Bluetooth integration is set up, and it
+discover a new device. In this case, it discovered a Mi Flora Flower
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
discovered a new device. In this case, it discovered a Mi Flora Flower
```

File: source/_posts/2022-08-03-release-20228.markdown [hunk 4]
```diff
@@ -0,0 +1,560 @@
...
+
+For example, if you have a Luxaflex cover, you can actually use it using the
+Hunter Douglas PowerView integration. And this is not the only example;
+there are many more! The problem is, how would one know that?
+
+To make setting up things in Home Assistant easier, integrations can now
+support multiple brands! So, if you now add an integration in Home Assistant
+and search for Luxaflex, you will find one!
+
+<img class="no-shadow" src='/images/blog/2022-08/supported-brands.png' alt='Screenshot demonstrating the supported brands; it is now to search for a brand that is supported by another integration'>
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
<img class="no-shadow" src='/images/blog/2022-08/supported-brands.png' alt='Screenshot demonstrating the supported brands; it is now possible to search for a brand that is supported by another integration'>
```

File: source/_posts/2022-08-03-release-20228.markdown [hunk 5]
```diff
@@ -0,0 +1,560 @@
...
+
+When you add a new integration that is supported by another integration,
+Home Assistant will tell you which integration provides the support and
+guides you to set up that integration.
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
guide you to set up that integration.
```

File: source/_posts/2022-08-03-release-20228.markdown [hunk 6]
```diff
@@ -0,0 +1,560 @@
...
+
+## Better maps colors
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]: Do we normally talk about maps in plural? The integration is called `map` right? Maybe consider changing it to say "map colors" and "the map integration"?

File: source/_posts/2022-08-03-release-20228.markdown [hunk 7]
```diff
@@ -0,0 +1,560 @@
...
+## Other noteworthy changes
+
+There is much more juice in this release; here are some of the other
+noteworthy changes this release:
+
+- "Z-Wave JS" is now shown and known as "Z-Wave" in Home Assistant.
+- The "System Health" setting menu item, has been moved to the "Repairs"
+  dashboard, in the overflow menu (three dots in the top right).
+- In the background, a started project to standardize the names entities and
+  devices should have. This work in progress will take a long time to complete,
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
- In the background, a project has been started to standardize the names of entities and
  devices. This work is in progress and will take a long time to complete,
```

File: source/_posts/2022-08-03-release-20228.markdown [hunk 8]
```diff
@@ -0,0 +1,560 @@
...
+  [@bdraco] and [@Djelibeybi]!
+- The [UniFi Network] now has update entities, allowing you to update the
+  firmware of UniFi devices directly from Home Assistant. Great addition [@J3173]!
+- The Material Design Icons have been updated to v7.0.96, giving you
+  [100](https://pictogrammers.github.io/@mdi/font/7.0.96/) brand new icons
+  to use 🤩 Thanks, [@goyney]!
+- [SwitchBot] now supports Hygrometers (thanks [@murtas]) and door/contact
+  sensors (thanks [@bdraco])!
+- [@Noltari] added update entities to the [QNAP QSW] integration! Nice!
+- [@frenck] added multifactor authentication support for [Verisure]. So, you
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
- [@frenck] added multifactor authentication support for [Verisure]. So now you
```

File: source/_posts/2022-08-03-release-20228.markdown [hunk 9]
```diff
@@ -0,0 +1,560 @@
...
+Our very active [Discord chat server](/join-chat) is an excellent place to be
+at, and don't forget to join our amazing [forums](https://community.home-assistant.io/).
+
+Found a bug or issue? Please report it in our [issue tracker](https://github.com/home-assistant/core/issues),
+to get it fixed! Or, check [our help page](/help) for guidance for more
+places you can go.
+
+Are you more into email? [Sign-up for our Building the Open Home Newsletter](/newsletter)
+to get the latest news about features, things happening in our community and
+[](https://github.com/home-assistant/core/pull/58768)other news about building an Open Home; straight into your inbox.
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
other news about building an Open Home; straight into your inbox.
```

File: source/_posts/2022-08-03-release-20228.markdown [hunk 10]
```diff
@@ -0,0 +1,560 @@
...
+Any automations or scripts that target the device will need to be corrected.
+
+([@Bre77] - [#75395]) ([documentation](/integrations/advantage_air))
+
+[@Bre77]: https://github.com/Bre77
+[#75395]: https://github.com/home-assistant/core/pull/75395
+
+---
+
+The previsouly deprecated `advantage_air.set_myzone` has been removed.
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
The previsouly deprecated service `advantage_air.set_myzone` has been removed.
```

File: source/_posts/2022-08-03-release-20228.markdown [hunk 11]
```diff
@@ -0,0 +1,560 @@
...
+([@bdraco] - [#74440]) ([documentation](/integrations/homekit_controller))
+
+[@bdraco]: https://github.com/bdraco
+[#74440]: https://github.com/home-assistant/core/pull/74440
+
+{% enddetails %}
+
+{% details "LIFX" %}
+
+Each LIFX device now has its integration entry. The existing entry will be
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
Each LIFX device now has its own integration entry. The existing entry will be
```

File: source/_posts/2022-08-03-release-20228.markdown [hunk 12]
```diff
@@ -0,0 +1,560 @@
...
+migrated to separate entries automatically as each device is discovered,
+and will automatically be removed once all devices are found.
+
+Migrating to a integration entry per device model allows Home Assistant to
```
Comment (MartinHjelmare) [inline source/_posts/2022-08-03-release-20228.markdown]:
```suggestion
Migrating to an integration entry per device allows Home Assistant to
```

File: source/_posts/2022-08-03-release-20228.markdown (hunk 6 above)
Comment (frenck) [reply source/_posts/2022-08-03-release-20228.markdown]: Right. It is not just the integration, but also cards (any map shown anywhere actually). I've adjusted the wording a little.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**